dashboard/app/models/block.rb

```diff
@@ -32,7 +32,7 @@ def write_block_files
     block_path = Rails.root.join "config/blocks/#{level_type}/#{name}.json"
     File.write block_path, file_json
 
-    return unless helper_code
+    return unless helper_code && !helper_code.empty?
```

Comment:
is this redundant?

Reply:
In ruby, the empty string is truthy.

I replaced this with just `helper_code.try(:present?)`.